### PR TITLE
Add rare-disease and AIP repos to all RD datasets

### PR DESCRIPTION
**Pull request by cassimons**

### CODE REVIEW

File: tokens/repository-map.json [hunk 1]
```diff
@@ -2,75 +2,96 @@
...
         "images",
         "joint-calling",
         "production-pipelines",
+        "rare-disease",
         "sv-workflows"
     ],
     "flinders-ophthal": [],
-    "gtex": [],
+    "gtex": [
+        "rare-disease"
```
Comment (illusional) [inline tokens/repository-map.json:43]: Is gtex a RD project?

File: tokens/repository-map.json [hunk 2]
```diff
@@ -118,8 +146,14 @@
     "tx-adapt": [
         "tx-adapt"
     ],
-    "udn-aus": [],
-    "udn-aus-training": [],
+    "udn-aus": [
+        "automated-interpretation-pipeline",
+
```
Comment (illusional) [inline tokens/repository-map.json:151]: Remove extra space:

```suggestion
```

**Review by illusional — APPROVED** [review]
Approved with minor feedback, let me know when you've merged and I'll release.

File: tokens/repository-map.json (hunk 1 above)
Comment (cassimons) [reply tokens/repository-map.json:43]: It is a reference dataset that RD projects will make extensive use of.

File: tokens/repository-map.json [hunk 3]
```diff
@@ -2,75 +2,96 @@
...
-        "automated-interpretation-pipeline"
+        "automated-interpretation-pipeline",
+        "rare-disease"
     ],
     "lof-curation": [
         "variant-curation-portal"
     ],
-    "mcri-lrp": [],
+    "mcri-lrp": [
+        "rare-disease"
```
[the remark this answers is not in the document]
Comment (cassimons) [reply tokens/repository-map.json:76]: Thanks.

**Comment by MattWellie** [comment]
At this point is it easier to add these two repos to the list with assumed permissions on all repos?

**Comment by illusional** [comment]
Maybe @lgruen could chime in, it is tedious to add for each new repo, unless you add like a tag to a dataset so it gets those repos added automatically.

**Comment by lgruen** [comment]
> Maybe @lgruen could chime in, it is tedious to add for each new repo, unless you add like a tag to a dataset so it gets those repos added automatically.

The workflow around this file isn't ideal for a number of reasons, but I'd hope this will get better once we can integrate this into @illusional upcoming "one stack" refactoring, i.e. deploy this automatically through Pulumi (and have more flexibility like tags too).

**Pull request merged**